caom2utils/caom2utils/caom2blueprint.py

```diff
@@ -3120,9 +3120,9 @@ def _try_range_return(self, index, lookup):
             f'Chunk.{lookup}.axis.range.end.val',
             index, _to_float, RefCoord)
         if aug_range_start and aug_range_end:
-            range = CoordRange1D(aug_range_start, aug_range_end)
-        self.logger.debug(f'Completed setting range for {lookup}')
-        return range
+            result = CoordRange1D(aug_range_start, aug_range_end)
+            self.logger.debug(f'Completed setting range with return for {lookup}')
```

Comment:
On purpose in the `if` block now?

Reply:
Yes. It helps with debugging, because if there's no log message, there's no input metadata.